R/between.R

```diff
@@ -4,13 +4,18 @@ between <- function(x,lower,upper,incbounds=TRUE,verbose=getOption("datatable.ve
   if (is.logical(lower)) lower = as.integer(lower)   # typically NA (which is logical type)
   if (is.logical(upper)) upper = as.integer(upper)   # typically NA (which is logical type)
   is_strictly_numeric <- function(x) is.numeric(x) && !inherits(x, "integer64")
```

Comment:
Couldn't you just change `is_strictly_numeric` to `inherits(x, "POSIXct") || (is.numeric(x) && inherits(x, "integer64"))` instead of the `unclass` stuff?

Reply:
@arunsrinivasan probably I could, but I got confused why int64 has been explicitly excluded from that, shouldn't it just work same as numeric? or unhandled int64 NA is a problem here?

Reply:
Because the code I wrote didn't take care of int64s.

Reply:
`unclass` removed, instead of `is_strictly_numeric` we now have
```r
is.supported = function(x) is.integer(x) || (storage.mode(x)=="double" && !inherits(x, "integer64"))
```
will try to add int64 support as AFAIK it should be only a matter of handling NAs.